jdaviz/configs/default/plugins/export/export.vue

```diff
@@ -207,6 +207,8 @@
       </v-row>
     </div>
 
+    <div style="display: grid; position: relative"> <!-- overlay container -->

```

Comment:
Not sure how much you want in the overlay container. This okay?

<img src="https://github.com/spacetelescope/jdaviz/assets/2090236/b1b5d5e7-1a70-4d4e-bdaf-f2c583903a09" width=300/>

Reply:
this is just over both the filename and the export button?  That looks perfect to me!